en/sql/datatype.rst

```diff
@@ -1141,7 +1141,7 @@ CHAR(n)
 
 A fixed-length character string is represented as **CHAR** *(n)*, in which *n* represents the number of characters. If *n* is not specified, the value is specified as 1, default value.
 
-When the length of a character string exceeds *n*, they are truncated. When character string which is shorter than *n* is stored, whitespace characters are used to fill up the trailing space.
+When the length of a character string exceeds *n*, they may be truncated on the insert/update operation if the allow_truncated_string configuration value is "yes" or error occurs if not. When character string which is shorter than *n* is stored, whitespace characters are used to fill up the trailing space.
```

Comment:
the excess characters may be truncated in..... (on-->in)
if the allow_truncated_string configuration value is "yes" otherwise an error occurs. (or...not --> otherwise, added 'an')